aws/templates/common.ftl

```diff
@@ -1891,61 +1891,9 @@
         } +
         attributeIfTrue("Instance", port.LB.Instance??, port.LB.Instance!"") +
         attributeIfTrue("Version",  port.LB.Version??, port.LB.Version!"") +
-        attributeIfContent("PortMapping",  portMapping)
+        attributeIfContent("PortMapping",  portMapping) + 
+        attributeIfContent("DestinationPort" , port.Name)
```

Comment:
Not convinced about this. We need to keep link semantics clean - Dest port is really an attribute of the thing the link points to, rather than something needed to follow the link